src/core/statistics/useStatistics.ts

```diff
@@ -0,0 +1,60 @@
+// Useful resources for Mapp tracking:
+// https://documentation.mapp.com/1.0/en/manual-track-request-25105181.html
+// https://documentation.mapp.com/1.0/en/how-to-send-manual-tracking-requests-page-updates-7240681.html
+
+export type CustomClickParameter = {
+  [key: number]:
+    | number
+    | { [key: number]: string }
+    | { [key: string]: string[] };
+};
```

Comment:
Note that is also possible to use the TS utility: `Record`:
```
 type CustomClickParameter = Record<
    string,
    number | Record<number, string> | Record<string, string[]>
  >;

Reply:
Yeah is that new? I keep seeing it since yesterday or two days ago. Or maybe I'm just late to the party 😝
Thanks though, will use it ;)